includes/tracks/events/class-wc-order-tracking.php

```diff
@@ -22,12 +22,15 @@ public function init() {
 	/**
 	 * Send a Tracks event when an order is viewed.
 	 *
-	 * @param object $order Order.
+	 * @param WC_Order $order Order.
 	 */
 	public function track_order_viewed( $order ) {
+		if ( ! $order->get_id() ) {
```

Comment:
Maybe also check if `$order` is not null just in case this gets called incorrectly?

Reply:
Added a check for `$order instanceof WC_Order `